docs/references/backend/overview.mdx

```diff
@@ -109,3 +208,24 @@ try {
   // Error handling
 }
 ```
+
+## `createClerkClient({ options })`
+
+The `createClerkClient()` function requires an `options` object. It is recommended to set these options as [environment variables](/docs/deployments/clerk-environment-variables#api-and-sdk-configuration) where possible, and then pass them to the function. For example, you can set the `secretKey` option using the `CLERK_SECRET_KEY` environment variable, and then pass it to the function like this: `createClerkClient({ secretKey: process.env.CLERK_SECRET_KEY })`.
+
+The following options are available:
+
+| Property | Type | Description |
+| --- | --- | --- |
+| `secretKey` (required) | `string` | The Clerk secret key from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page in the Clerk Dashboard. |
+| `jwtKey?`  | `string` | The PEM public key from the **[API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page -> Show JWT public key -> PEM Public Key** section of the Clerk Dashboard. For more information, refer to [Manual JWT verification](/docs/backend-requests/handling/manual-jwt). |
+| `publishableKey?` | `string` | The Clerk publishable key from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page in the Clerk Dashboard. |
+| `domain?` | `string` | The domain for the application. For development, you can pass the `localhost` your application is running on. For example: `localhost:3001`. |
```

Comment:
@panteliselef Could you verify if the `domain` options is only used int multi-domain setup?

Reply:
Correct, Only in a multi-domain setup, and only for the satellite apps, not the one that is used as the primary.

And i believe this is only needed for handshake to work, so for people that use `@clerk/backend` in order simply to access our Backend API it will not make a difference if they use `domain` or even `proxyUrl`.